compiler-plugin-test/src/test/resources/SingleAggregateInLoop.template.kt

```diff
@@ -0,0 +1,20 @@
+import it.unibo.collektive.aggregate.api.Aggregate
+%(imports)
+
+fun Aggregate<Int>.exampleAggregate(): Int = 0
+
+/**
+    Kotlin program template with four template lines: before/after the loop and
+    before/after the aggregate call inside the loop.
+    The parts after "%" are placeholders for string formatting that allow putting an arbitrary
+    piece of code inside them (avoiding having to create multiple testing sources).
+*/
+fun Aggregate<Int>.x(pivot: (Int) -> Any?) {
```

Comment:
Why do you specify the `pivot` as the input argument?

Reply:
In that case, the `localId` divides devices based on their ID (even and odd).
In this case, I would avoid having the pivot based on the `localId`

Reply:
Ok, will fix it in a moment.